src/AEG/AtomNode.ts

```diff
@@ -157,7 +157,7 @@ export class AtomNode {
      */
     public calcRect(): Rectangle {
         return new Rectangle(
-            new Point(this.internalOrigin.x, this.internalOrigin.y - this.internalHeight),
+            new Point(this.internalOrigin.x, this.internalOrigin.y),
```

Comment:
I personally preferred the mouse at the bottom of the atom

Reply:
Determined not as important as keeping tests running on 07/26/2024 at 7:55 PM.